src/stories/components/cu-table-header-skeleton/header-skeleton.tsx

```diff
@@ -18,13 +20,13 @@ export const CuTableHeaderSkeleton = () => {
       />
       <Filters>
         <Skeleton
+          className="hello"
```

Comment:
sorry i don't understand what is this class for, can you give me a cuvee, for what its

Reply:
It was a mistake, removed